lib/parallel.rb

```diff
@@ -697,9 +697,10 @@ def instrument_start(item, index, options)
     end
 
     def available_processor_count
+      gem 'concurrent-ruby', '>= 1.3.4'
```

Comment:
What does this raise if ran under bundle exec with an older `concurrent-ruby`?

Reply:
`Gem::LoadError` will be raised. `Gem::LoadError` is a descendant of `LoadError`. So we can rescue it in https://github.com/grosser/parallel/blob/ab06d9e6d79a8b1375debfc348ecc19cd54e390e/lib/parallel.rb#L702